lib/internal/process/next_tick.js

```diff
@@ -116,4 +116,7 @@ function setupNextTick(_setupNextTick, _setupPromises) {
       tickInfo[kHasScheduled] = 1;
     queue.push(new TickObject(callback, args, getDefaultTriggerAsyncId()));
   }
+
+  process.stillSynchronous = true;
```

Comment:
This shouldn't expose undocumented things directly on `process` — that will change.